plans/dht/test/find_peers.go

```diff
@@ -9,46 +9,39 @@ import (
 	"github.com/ipfs/testground/sdk/runtime"
 	"github.com/ipfs/testground/sdk/sync"
 
-	"github.com/libp2p/go-libp2p"
-	"github.com/libp2p/go-libp2p-core/host"
-	"github.com/libp2p/go-libp2p-core/peer"
-	dht "github.com/libp2p/go-libp2p-kad-dht"
-	dhtopts "github.com/libp2p/go-libp2p-kad-dht/opts"
-
-	"github.com/ipfs/go-datastore"
+	utils "github.com/ipfs/testground/plans/dht/utils"
+	host "github.com/libp2p/go-libp2p-core/host"
+	peer "github.com/libp2p/go-libp2p-core/peer"
 )
 
-func LookupPeers(runenv *runtime.RunEnv) {
+// NOTE: Needs to run with latest kad-dht. Use:
+// go build . && ./testground run dht/find-peers --builder=docker:go --runner="local:docker" --dep="github.com/libp2p/go-libp2p-kad-dht=master"
+
+// FindPeers is the Find Peers Test Case
+func FindPeers(runenv *runtime.RunEnv) {
+	// Test Parameters
 	var (
-		timeout    = time.Duration(runenv.IntParamD("timeout_secs", 30)) * time.Second
-		bucketSize = runenv.IntParamD("bucket_size", 20)
+		timeout = time.Duration(runenv.IntParamD("timeout_secs", 30)) * time.Second
+		// nFindPeers = runenv.IntParamD("n_find_peers", 10)
 	)
 
+	/// --- Warm up
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
 
-	// moot assignment to pacify the compiler. We need to merge the configurable
-	// bucket size param upstream.
-	_ = bucketSize
-
-	h, err := libp2p.New(context.Background())
+	node, dht, err := utils.CreateDhtNode(ctx, runenv)
```

Comment:
`utils` packages are rather unidiomatic in Go: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Consider renaming the package to `dhtt` (dht test) and the function to `CreateNode()`, e.g.

```
node, dht, err := dhtt.CreateNode(ctx, runenv)
```

Reply:
Gotcha. I listened to a podcast that also touched on this but I was unsure how best organize things in this case. What I have is an attempt to follow our discussion about separating the "set up" from "test case"

<img width="370" alt="image" src="https://user-images.githubusercontent.com/1211152/68292515-f2a77d00-0083-11ea-9bff-e6ba91235cc8.png">

Is `s/utils/dhtt` really more informative? Perhaps the trick is to just put this create-dht-node.go at the /dht folder level